code/modules/chemistry/Chemistry-Dispenser.dm

```diff
@@ -34,6 +36,7 @@
 
 	New()
 		..()
+		src.dispensable_reagents = basic_elements
```

Comment:
this will force all subtypes of `/obj/machinery/chem_dispenser` (e.g.: soda dispenser) to use this list as well

Reply:
whoops